doc/api/next_api_changes/deprecations/22268-OG.rst

```diff
@@ -0,0 +1,6 @@
+``ticker`` functions deprecated
+-------------------------------
+
+The functions ``matplotlib.ticker.is_decade`` and
+``matplotlib.ticker.is_close_to_int`` are considered internal and public access
```

Comment:
Can you add a note about what user should use instead?

Reply:
Should they use anything instead? Like `y = math.log(x)/math.log(base); math.isclose(y, round(y))`? With the numpy alternative as well?

(Interestingly it seems like all the code assumes scalars since there are no `all` anywhere that I have seen, so maybe one should just go with the `math` version although `is_scalar` did support arrays.)

(I missed this comment before pushing the change for the previous comment... But will add something if it is a bit more clear what should be added. It just seems like a quite long piece of code for `is_scalar`, `is_close_to_integer` is more straightforward.)